league_rpc_linux/processes/lcu_thread.py

```diff
@@ -0,0 +1,377 @@
+import time
+from threading import Timer
+from dataclasses import dataclass
+
+import lcu_driver
+from lcu_driver.connection import Connection
+from lcu_driver.events.responses import WebsocketEventResponse
+from pypresence import Presence
+
+from league_rpc_linux.colors import Colors
+from league_rpc_linux.const import (
+    BASE_MAP_ICON_URL,
+    GAME_MODE_CONVERT_MAP,
+    LEAGUE_OF_LEGENDS_LOGO,
+    MAP_ICON_CONVERT_MAP,
+    PROFILE_ICON_BASE_URL,
+    SMALL_TEXT,
+)
+
+
+@dataclass
+class ClientData:
+    availability: str = "Online"  # "Online", "Away
+    gamemode: str = None
```

Comment:
Incompatible type hinting.

You have type hinted this value as to be a string, but you are assigning a default value of "None".

The correct type hint in this case would be ``gamemode: Optional[str] = None``

where ``Optional`` is imported from ``typing`` module.

Alternatively what you can do is to set a falsey value..

so ``gamemode: str = ""`` This will work.. however you need to take that into account if you have used ``if clientdata.gamemode is not None`` somewhere.. which i don't think you have.

Reply:
then again.. this comment applies to all variables you have typehinted as a specific type, but used "None" as it's default.

Reply:
This has been fixed in the latest commit.